packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2484,6 +2484,16 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   @override
   void userUpdateTextEditingValue(TextEditingValue value, SelectionChangedCause? cause) {
+    // TODO(chunhtai): Figure out why context or renderEditable can be null
+    // https://github.com/flutter/flutter/issues/79818.
+    if (_editableKey.currentContext == null || _editableKey.currentContext!.findRenderObject() == null) {
```

Comment:
Nit: Does something like this work? `_editableKey.currentContext?.findRenderObject() == null`